esrecurse.js

```diff
@@ -53,8 +55,13 @@
         return (nodeType === estraverse.Syntax.ObjectExpression || nodeType === estraverse.Syntax.ObjectPattern) && key === 'properties';
     }
 
-    function Visitor(visitor) {
+    function Visitor(visitor, options) {
+        options = options || {};
+
         this.__visitor = visitor ||  this;
+        this.__childVisitorKeys = options.childVisitorKeys
```

Comment:
`esrecurse` is not using `estraverse` in order to traverse AST, so we cannot use the `keys` option of `estraverse` here.